cmd/create/create.go

```diff
@@ -0,0 +1,77 @@
+package create
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"realcloud.tech/pligos/cmd/compile"
+	"realcloud.tech/pligos/pkg/helm"
+	"realcloud.tech/pligos/pkg/pligos"
+)
+
+type Config struct {
+	ConfigPath  string
+	ContextName string
+	ChartPath   string
+}
+
+func Create(config Config) error {
+	return createTree(config.ConfigPath, config.ContextName, config.ChartPath)
+}
+
+func NewHelm(configPath, contextName string) (*helm.Helm, error) {
+	config, err := pligos.OpenPligosConfig(configPath)
+	if err != nil {
+		return nil, err
+	}
+
+	c, err := compile.NewCompiler(configPath, contextName)
+	if err != nil {
+		return nil, err
+	}
+
+	context, ok := pligos.FindContext(contextName, config.Contexts)
+	if !ok {
+		return nil, fmt.Errorf("no such context: %s", contextName)
+	}
+	return helm.New(context.Flavor, config.DeploymentConfig, config.ChartDependencies, context.Configs, context.Secrets, c), nil
+}
+
+func createTree(configPath, contextName, chartPath string) error {
```

Comment:
This method could go to pkg/pligos, maybe?
If we need a struct to represent the chart tree, then we should have this abstraction explicitly declared, a go file to hold this, maybe a struct (needed?) and this method inside the file.